src/lib/utils/searchResults.ts

```diff
@@ -0,0 +1,17 @@
+//- Library Improts
+import { DisplayParentDomain } from 'lib/types';
+
+export const filterSearchResultsByNetwork = (
+	results: DisplayParentDomain[],
```

Comment:
@domw30  What is the purpose of this util?

Reply:
@padala, the intention was to filter domains from another network when ```REACT_APP_NETWORK=wilder```. Say for example the network is set to ```tester```, when searching the results will show domains on ```wilder``` network. If selected, you'd hit a page not found. I thought now we switch between networks this may be useful? Let me know your thoughts and feedback, much appreciated Subbu thanks.

Reply:
@domw30 could work out nicer to append the root network to any search queries if there is one - that way we don't have to do any additional filtering in the UI.

Reply:
@colbr, do you mean append the root network as it currently is? 

<img width="832" alt="Screenshot 2022-07-05 at 17 24 30" src="https://user-images.githubusercontent.com/39112648/177373485-b72fed82-8b58-4ac5-936c-4e7202d9cf13.png">

Reply:
Hmm, on second thought, doing it your way makes more sense. Appending the root network to the search query would prevent people from searching "wilder.beasts" (as the query would be "wilder.wilder.beasts"). I'll review the rest of this one today, just saw this comment as I was scrolling and thought I'd chime in!

Reply:
Thanks mate! 🚀